hack/README.md

```diff
@@ -0,0 +1,69 @@
+docker/hack
+============================
+
+## About
+
+`hack` contains a collection of scripts used by Docker. If there are any issues regarding the intention of a particular script
```

Comment:
Not sure if we should change the usage of `Docker` to `Moby` here, since Docker is still a package put together by Moby. Would we want to raise a separate issue surrounding this if it hasn't been raised already?